uxarray/grid.py

```diff
@@ -225,3 +227,34 @@ def __init_ds_var_names__(self):
             "nMesh2_face": "nMesh2_face",
             "nMaxMesh2_face_nodes": "nMaxMesh2_face_nodes"
         }
+
+    def __init_ds_var_methods__(self):
+        """Initialize variables for directly accessing Coordinate and
+        Data variables through ugrid conventions
+
+        Examples
+        ----------
+        Initialize some Grid Object
+        >>> grid = ux.open_dataset(file_path)
+
+        Return Xarray Data Array
+        >>> x = grid.Mesh2_node_x      
+
+        Return Numpy Array
```

Comment:
This is (i.e. Xarray data array to Numpy array conversion) beyond the scope of this example and function I believe.